rewrite-java/src/main/java/org/openrewrite/java/AddOrUpdateAnnotationAttribute.java

```diff
@@ -255,4 +282,23 @@ private static boolean attributeIsString(@Nullable String attributeName, J.Annot
         }
         return false;
     }
+
+    private static boolean attributeValIsAlreadyPresentOrNull(List<Expression> expression, @Nullable String attributeValue) {
```

Comment:
What does the `OrNull` mean in the method name suffix here? Because I do not see this return `true` if the new or old attribute value were to be `null`.